feature-libs/organization/administration/components/shared/list/list.service.ts

```diff
@@ -158,4 +158,24 @@ export abstract class ListService<T, P = PaginationModel> {
     pagination: PaginationModel,
     ...args: any
   ): Observable<EntitiesModel<T> | undefined>;
+
+  /**
+   * If this method returns true, hyperlink will be shown to add new user.
+   * If this method returns false, button will be displayed.
+   */
+  showLink(): boolean {
+    return true;
+  }
```

Comment:
As @armanozak mentioned, let's not return a boolean, but something more generic like `enum` value (so to be prepared for possible new extensions in the future, if only needed).

e.g.
```ts
export enum ListItemCreateUIType {
  LINK = 'LINK';
  BUTTON = 'BUTTON'
}
```

```ts
getCreateButtonType(): ListItemCreateUIType {
  return ListItemCreateUIType.LINK;
} 
```

Reply:
done